code/datums/config/config_num.dm

```diff
@@ -0,0 +1,454 @@
+
+/decl/config/num/health_stress_blood_recovery_constant
+	uid = "stress_blood_recovery_constant"
+	default_value = 0.3
+	config_description = "A multiplier for the impact stress has on blood regeneration, as above."
+
+/decl/config/num/health_health_threshold_dead
+	uid = "health_threshold_dead"
+	default_value = -100
+	config_description = "Level of health at which a mob becomes dead."
+
+/decl/config/num/health_organ_health_multiplier
+	uid = "organ_health_multiplier"
+	default_value = 0.9
+	config_description = list(
+		"Multiplier which enables organs to take more damage before bones breaking or limbs being destroyed.",
+		"100 means normal, 50 means half."
+	)
+
+/decl/config/num/health_organ_health_multiplier/set_value(var/new_value)
+	new_value /= 100
+	return ..(new_value)
+
+/decl/config/num/health_organ_regeneration_multiplier
+	uid = "organ_regeneration_multiplier"
+	default_value = 0.25
+	config_description = list(
+		"Multiplier which influences how fast organs regenerate naturally.",
+		"100 means normal, 50 means half."
+	)
+
+/decl/config/num/health_organ_regeneration_multiplier/set_value(var/new_value)
+	new_value /= 100
+	return ..(new_value)
+
+/decl/config/num/health_organ_damage_spillover_multiplier
+	uid = "organ_damage_spillover_multiplier"
+	default_value = 0.5
+
+/decl/config/num/health_organ_damage_spillover_multiplier/set_value(var/new_value)
+	new_value /= 100
+	return ..(new_value)
+
+/decl/config/num/health_revival_brain_life
+	uid = "revival_brain_life"
+	default_value = -1
+	config_description = "Amount of time (in hundredths of seconds) for which a brain retains the 'spark of life' after the person's death (set to -1 for infinite)."
+
+//force disconnect for inactive players after this many minutes, if non-0
+/decl/config/num/kick_inactive
+	uid = "kick_inactive"
+	config_description = "Disconnect players who did nothing during the set amount of minutes."
+
+/decl/config/num/fps
+	uid = "fps"
+	default_value = 20
+	config_description = list(
+		"Defines world FPS. Defaults to 20.",
+		"Can also accept ticklag values (0.9, 0.5, etc) which will automatically be converted to FPS."
+	)
+
+/decl/config/num/fps/sanitize_value()
+	value = round(value)
+	if(value <= 0)
+		value = default_value
+	. = ..()
+
+// Handle ticklag-formatted FPS (0.7 etc)
+/decl/config/num/fps/set_value(new_value)
+	if(new_value > 0 && new_value < 1)
+		new_value = round(10 / new_value)
+	return ..(new_value)
+
+/decl/config/num/fps/update_post_value_set()
+	world.fps = value
+	. = ..()
+
+//SSinitialization throttling
+/decl/config/num/tick_limit_mc_init
+	uid = "tick_limit_mc_init"
+	default_value = TICK_LIMIT_MC_INIT_DEFAULT
+
+//if objectives are disabled or not
+/decl/config/num/objectives_disabled
+	uid = "objectives_disabled"
+	default_value = CONFIG_OBJECTIVE_NONE
+	config_description = "If objectives are disabled."
+
+/decl/config/num/objectives_disabled/set_value(new_value)
+	switch(new_value)
```

Comment:
Like I mentioned earlier maybe making an enum subtype for this kind of value be worthwhile?
Also, it's currently a bit ambiguous cause it's labelled as a /decl/num but it parses a string. It might be a good idea to explain somewhere that the default value and the /decl/config subtype name is based on the stored value and not the one inside the config file?